srcpkgs/spacenavd/files/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
```

Comment:
```suggestion
exec 2>&1
```

Reply:
Fixed in https://github.com/void-linux/void-packages/commit/9cc48924ccc11c7e2ac591643aeaddce28230987